python/cuspatial/cuspatial/_lib/spatial.pyx

```diff
@@ -84,31 +91,31 @@ cpdef cpp_haversine_distance(x1, y1, x2, y2):
 
     return result
 
-cpdef cpp_lonlat2coord(cam_lon, cam_lat, in_lon, in_lat):
-    cam_lon = np.float64(cam_lon)
-    cam_lat = np.float64(cam_lat)
-    in_lon = in_lon.astype('float64')._column
-    in_lat = in_lat.astype('float64')._column
-    cdef gdf_scalar* c_cam_lon = gdf_scalar_from_scalar(cam_lon)
-    cdef gdf_scalar* c_cam_lat = gdf_scalar_from_scalar(cam_lat)
-    cdef gdf_column* c_in_lon = column_view_from_column(in_lon)
-    cdef gdf_column* c_in_lat = column_view_from_column(in_lat)
+def lonlat_to_cartesian(
+    origin_lon,
+    origin_lat,
```

Comment:
You should be able to just make these `double origin_loin` and `double origin_lat` and avoid needing to create a new cdef for them below.